ecommerce/models.py

```diff
@@ -322,6 +322,12 @@ class Order(OrderAbstract, AuditableModel):
         settings.AUTH_USER_MODEL, on_delete=models.PROTECT, related_name="orders"
     )
     total_price_paid = models.DecimalField(decimal_places=2, max_digits=20)
+    # These represent the tax collected for the entire order.
+    tax_country_code = models.CharField(max_length=2, blank=True, null=True)
+    tax_rate = models.DecimalField(
+        max_digits=6, decimal_places=4, null=True, blank=True, default=0
+    )
+    tax_rate_name = models.CharField(max_length=100, null=True, default="VAT")
```

Comment:
Yes, that was the idea. Tax rates change (not often, but they do) so it made sense to add the rate charged at the time to the order. (In addition, I'm slightly hedging about the determination that was made to charge tax for the order - I wanted to store the determined country in case of audit.)